packages/dd-trace/src/appsec/reporter.js

```diff
@@ -151,8 +181,11 @@ module.exports = {
   metricsQueue,
   filterHeaders,
   formatHeaderName,
+  reportWafInit,
   reportMetrics,
   reportAttack,
+  reportBlock,
+  reportWafUpdate,
```

Comment:
instead of having a passthrough function, you could just do that ?
```suggestion
  reportWafUpdate: incrementWafUpdatesMetric,
```